network/transport/v2/handlers.go

```diff
@@ -244,7 +244,13 @@ func (p *protocol) handleTransactionList(peer transport.Peer, envelope *Envelope
 		}
 		if !present {
 			// TODO does this always trigger fetching missing payloads? (through observer on DAG) Prolly not for v2
-			if err = p.state.Add(ctx, tx, envelope.TransactionList.Transactions[i].Payload); err != nil {
+			err = p.state.Add(ctx, tx, envelope.TransactionList.Transactions[i].Payload)
+			if err != nil {
+				if errors.Is(err, dag.ErrPreviousTransactionMissing) {
+					p.cMan.done(cid)
```

Comment:
If the peer sent a chunked list, it causes errors. Do we expect this situation to occur often? Then the error message (TransacitonList for finished conversation) could be cumbersome.

Reply:
Chunked lists should still send all messages in the correct order. A Tx with a missing prevs cannot be processed, and Txs received in later chucks are likely to depend on the Tx that is skipped here. This results in a chain of Txs that cannot be processed. 

I'd prefer to cancel the request at this point, but ignoring all future messages is the closest we have right now.

Reply:
Added a logger to keep track of this